packages/common/src/downloads/types.ts

```diff
@@ -104,6 +104,15 @@ export interface IStaticDownloadFormat extends IDownloadFormat {
   url: string;
 }
 
+/**
+ * Extends the typical static download format, and is used
+ * by `toStaticFormat` in `packages/common/src/downloads/fetchDownloadFormats.ts`
+ */
+export interface IAdditionalResourceDownloadFormat
+  extends IStaticDownloadFormat {
+  isDataSource: boolean;
+}
+
```

Comment:
Confirmation question: I don't need to extend the dynamic download format, do I?